### PR TITLE
Code Quality

### PR DESCRIPTION
**Pull request by johnnyvuanly**

### CODE REVIEW

**Comment by claraj** [comment]
@GabrielGramlich can you review?

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,52 @@
...
         data = json.loads(data)
 
-    print(data)
+    # print(data)
     
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    state_Abbr = data # dictionary of state abbreviations keys and state name values 
+    state_Abbr2 = {} # dictionary of state name keys and state abbreviation values
```
Comment (GabrielGramlich) [inline convert_state.py]: While you've made this snake case, the "Abbr" should still be all lowercase, and the state_abbr and state_abbr2 should be more distinct, perhaps abbreviations and states instead.

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,52 @@
...
+
+def convert_state_to_abbreviation(dictionary):
+    """ Allow user to get conversion from dictionary """
+    user_input = input('Enter state name').capitalize()
+    result = dictionary.get(user_input)
+    if result is None:
+        print('State not found')
     else:
-        print ( "The abbreviation for " +  userInput+" is " + result)
+        print ('The abbreviation for' +  user_input + ' is ' + result)
```
Comment (GabrielGramlich) [inline convert_state.py:42]: While I don't think the linter catches this, I'd recommend changing these to an f string, as there's more than one variable being added to the string.

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,52 @@
...
 def convert_abbreviation_to_state(dictionary):
-    userInput   = input("Enter abbrviation name").upper()
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
+    """ Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states """
+    user_input = input('Enter abbreviation name').upper()
+    result = dictionary.get(user_input)
+    if result is None:
+        print('Abreviation not found')
```
Comment (GabrielGramlich) [inline convert_state.py:49]: There's a typo in "Abreviation" here. Should be two b's.

File: convert_state.py [hunk 4]
```diff
@@ -8,11 +9,11 @@ def main():
...
     
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data # dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {} # dictionary of state name keys and state abbreviation values
+    state_Abbr = data # dictionary of state abbreviations keys and state name values 
+    state_Abbr2 = {} # dictionary of state name keys and state abbreviation values
     # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
+    for statekey, statevalue in state_Abbr.items() :
```
Comment (GabrielGramlich) [inline convert_state.py]: statekey, statevalue aren't really meaningful. Which is which?

File: convert_state.py [hunk 5]
```diff
@@ -1,60 +1,54 @@
+""" Program to convert state names to abbreviation and vice versa """
+import json
 
-
-
-import json  # import json for json processing
 def main():
+    """ Allow user to get conversion from dictionary """
     with open('data.json','r') as file:
```
Comment (claraj) [inline convert_state.py:6]: put a space after the commas in arguments

File: convert_state.py [hunk 6]
```diff
@@ -1,60 +1,54 @@
...
         data = file.read()
         data = json.loads(data)
 
-    print(data)
+    # print(data)
```
Comment (claraj) [inline convert_state.py:10]: Remove this line

File: convert_state.py [hunk 7]
```diff
@@ -1,60 +1,54 @@
...
     
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    state_abbr = data # dictionary of state abbreviations keys and state name values
```
Comment (claraj) [inline convert_state.py:13]: Can you think of a better variable name?

File: convert_state.py [hunk 8]
```diff
@@ -1,60 +1,54 @@
...
         data = json.loads(data)
 
-    print(data)
+    # print(data)
     
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    state_abbr = data # dictionary of state abbreviations keys and state name values 
+    state_abbr2 = {} # dictionary of state name keys and state abbreviation values
```
Comment (claraj) [inline convert_state.py:14]: Can you think of a better variable name?

File: convert_state.py [hunk 9]
```diff
@@ -1,60 +1,54 @@
...
-
-        if choice=="1":
-
-            convertStateToAbbreviation(stateAbbr2)
-        elif  choice == "2":
-            convert_abbreviation_to_state(stateAbbr)
-        
-        
-        elif choice==  "3":
+        print('2. Convery abbreviation to state')
```
Comment (claraj) [inline convert_state.py:21]: Fix spelling "Convert"

File: convert_state.py [hunk 10]
```diff
@@ -1,60 +1,54 @@
...
+        print('3. quit')
+        choice = input('Enter choice: ')
+
+        if choice=='1':
+            convert_state_to_abbreviation(state_abbr2)
+        elif  choice == '2':
+            convert_abbreviation_to_state(state_abbr)
+        elif choice==  '3':
```
Comment (claraj) [inline convert_state.py:29]: Fix spacing around operators

**Review by claraj — CHANGES_REQUESTED** [review]
Looking much better, still some issues to fix

**Pull request closed without merging**